test/nbrowser/AccessRules2.ts

```diff
@@ -109,6 +109,7 @@ describe("AccessRules2", function() {
     assert.equal(await driver.findWait('.test-view-as-banner', 2000).isPresent(), true);
     assert.match(await driver.find('.test-view-as-banner .test-select-open').getText(),
                  new RegExp(gu.translateUser('user3').name, 'i'));
+    assert.equal(await driver.find('.test-info-tooltip.test-view-as-help-tooltip').isPresent(), true);
```

Comment:
You can change `isPresent` to `isDisplayed` to make this check a tiny bit stricter: `isPresent` only checks if an element is present in the DOM, while `isDisplayed` checks that it's visible (e.g. doesn't have `display: none` style).